### PR TITLE
Making knifes able to accept chemicals like spear.

### PR DESCRIPTION
**Pull request by Xeno4**
Now you can pump some unstable mutagen into the knife and do the funny stuff.
image

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/knife.yml [hunk 1]
```diff
@@ -64,7 +76,7 @@
   - type: Item
     size: 10
     sprite: Objects/Weapons/Melee/cleaver.rsi
-
+
```
Comment (0x6273) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/knife.yml:79]: Whitespace. Also on line 104.

**Comment by asperger-sind** [comment]
These probably shouldn't be able to hold 5u, maybe less, like 3u.

**Comment by Xeno4** [comment]
> These probably shouldn't be able to hold 5u, maybe less, like 3u.

5U is really low since it transfers only 1u per hit, that means you can only do 30 radiation damage(if you use unstable mutagen) by 5 hits. Also if you use lexorin it wont kill the target immediately so its not that op

**Comment by asperger-sind** [comment]
> 5U is really low since it transfers only 1u per hit, that means you can only do 30 radiation damage (if you use unstable mutagen) by 5 hits. Also, if you use lexorin it won't kill the target immediately so it's not that op

The issue here is that spears take up 40 storage, and to be used effectively require to be wielded, knives are small at 10 storage, attack 1.5x faster without the need to be wielded, and to all that you can still just keep a beaker in your 2nd hand and instantly refill the knife once it's dry, which the spear cannot offer, and you still can just use something like licoxide to flat out stunlock someone with a tipped knife.

As a sidenote, I don't think knives should even have the ability to contain solutions inside of them.

**Comment by Xeno4** [comment]
> > 5U is really low since it transfers only 1u per hit, that means you can only do 30 radiation damage (if you use unstable mutagen) by 5 hits. Also, if you use lexorin it won't kill the target immediately so it's not that op
> 
> The issue here is that spears take up 40 storage, and to be used effectively require to be wielded, knives are small at 10 storage, attack 1.5x faster without the need to be wielded, and to all that you can still just keep a beaker in your 2nd hand and instantly refill the knife once it's dry, which the spear cannot offer, and you still can just use something like licoxide to flat out stunlock someone with a tipped knife.
> 
> As a sidenote, I don't think knives should even have the ability to contain solutions inside of them.
Yea, this makes a point, ill reduce the storage of chemicals. Knifes containing a solution makes them better in fights.

**Comment by mirrorcult** [comment]
this is a no go for any weapon that isn't wieldable

**Comment by Kromabae** [comment]
i'd say no to this since it's very easy for a botanist to get a knife and very easy for them to get amatoxin (which is considered worse than lexorin now since lexorin got a balance rework)

5u amatoxin means after 5 hits you've done an extra 60 damage ish

**Comment by Xeno4** [comment]
> i'd say no to this since it's very easy for a botanist to get a knife and very easy for them to get amatoxin (which is considered worse than lexorin now since lexorin got a balance rework)
> 
> 5u amatoxin means after 5 hits you've done an extra 60 damage ish

The solution has been changed to 3u

**Comment by OldDanceJacket** [comment]
> this is a no go for any weapon that isn't wieldable

Why is that? Can't the scalpel in ss13 also have chems put on it?

**Comment by mirrorcult** [comment]
> > this is a no go for any weapon that isn't wieldable
> 
> Why is that? Can't the scalpel in ss13 also have chems put on it?
> 
> Also why only weapons that are wieldable? Isn't the point of wielding a weapon is that you unlock new actions and higher damage at the cost of a do-after and restricting both of your hands to hold it?

because you cant put a beaker in your offhand to instantly refill it if you're wielding it

**Comment by OldDanceJacket** [comment]
> > > this is a no go for any weapon that isn't wieldable
> > 
> > 
> > Why is that? Can't the scalpel in ss13 also have chems put on it?
> > Also why only weapons that are wieldable? Isn't the point of wielding a weapon is that you unlock new actions and higher damage at the cost of a do-after and restricting both of your hands to hold it?
> 
> because you cant put a beaker in your offhand to instantly refill it if you're wielding it

But... You can. You can just use the spear one-handed no? Is it set to only interact with chems when using both hands? Also why does that matter?

**Comment by Xeno4** [comment]
> > > this is a no go for any weapon that isn't wieldable
> > 
> > 
> > Why is that? Can't the scalpel in ss13 also have chems put on it?
> > Also why only weapons that are wieldable? Isn't the point of wielding a weapon is that you unlock new actions and higher damage at the cost of a do-after and restricting both of your hands to hold it?
> 
> because you cant put a beaker in your offhand to instantly refill it if you're wielding it

I mean spear still does like 12 damage when not wielded and + it has higher capacity than the knife(i changed the knife to 3u instead of 5).

**Comment by Xeno4** [comment]
Its able to inject chems even single handed

**Comment by mirrorcult** [comment]
> > > > this is a no go for any weapon that isn't wieldable
> > > 
> > > 
> > > Why is that? Can't the scalpel in ss13 also have chems put on it?
> > > Also why only weapons that are wieldable? Isn't the point of wielding a weapon is that you unlock new actions and higher damage at the cost of a do-after and restricting both of your hands to hold it?
> > 
> > 
> > because you cant put a beaker in your offhand to instantly refill it if you're wielding it
> 
> But... You can. You can just use the spear one-handed no? Is it set to only interact with chems when using both hands? Also why does that matter?

because the spear is significantly worse if its not wielded with low damage & attack speed

**Comment by mirrorcult** [comment]
it honestly should just get removed from the spear too but i'd rather not extend it to anything that would make it even better

**Comment by OldDanceJacket** [comment]
Honestly I disagree.

The spear _shouldn't_ have continued nerfs and it's chem function shouldn't be removed; Infact it should be added to the knife **and** scalpel and more interesting ways to take combat should be continued to be encouraged. 

Even still I think better/different alternative attacks should be put onto melee weapons **_AND_** I'd even say the spear should be buffed to have extra range on it's alternative attack which currently is rubbish.

**Comment by Xeno4** [comment]
> storage, attack 1.5x faster without the need to be wielded, and to all that you can still just keep a beaker in your 2nd hand

Some people still say that this would be too op to have knirfes get a little bit more damage

**Comment by Vordenburg** [comment]
> > > this is a no go for any weapon that isn't wieldable
> > 
> > 
> > Why is that? Can't the scalpel in ss13 also have chems put on it?
> > Also why only weapons that are wieldable? Isn't the point of wielding a weapon is that you unlock new actions and higher damage at the cost of a do-after and restricting both of your hands to hold it?
> 
> because you cant put a beaker in your offhand to instantly refill it if you're wielding it

A DoAfter for applying solutions to weapons would solve that. Make the user accidentally cut and inject themself if they're interrupted.

**Comment by Xeno4** [comment]
> > > > this is a no go for any weapon that isn't wieldable
> > > 
> > > 
> > > Why is that? Can't the scalpel in ss13 also have chems put on it?
> > > Also why only weapons that are wieldable? Isn't the point of wielding a weapon is that you unlock new actions and higher damage at the cost of a do-after and restricting both of your hands to hold it?
> > 
> > 
> > because you cant put a beaker in your offhand to instantly refill it if you're wielding it
> 
> A DoAfter for applying solutions to weapons would solve that. Make the user accidentally cut and inject themself if they're interrupted.

The solution gets applied instantly

**Comment by metalgearsloth** [comment]
Agree with mirror.

**Pull request closed without merging**